_slides/04-JamesBeckley-eaton.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+So what to I want to add here...
```

Comment:
This is a good start, but i think you should make it sound more like an abominable snowman's roar.